src/NuGet.Core/NuGet.Packaging/Signing/Verification/SignatureTrustAndValidityVerificationProvider.cs

```diff
@@ -175,6 +171,10 @@ private SignatureVerificationStatus VerifySignature(Signature signature, Timesta
                         issues.Add(SignatureLog.DebugLog(string.Format(CultureInfo.CurrentCulture, Strings.ErrorInvalidCertificateChain, string.Join(", ", chainStatuses.Select(x => x.ToString())))));
                     }
                 }
+                else
+                {
+                    issues.Add(SignatureLog.Issue(treatIssuesAsErrors, NuGetLogCode.NU3003, Strings.SignatureNotTimeValid));
```

Comment:
I think this should be NU3011.  NU3003 would be if a package didn't verify successfully if the package wasn't signed with a certificate indicated by the `CertificateFingerprint` option.